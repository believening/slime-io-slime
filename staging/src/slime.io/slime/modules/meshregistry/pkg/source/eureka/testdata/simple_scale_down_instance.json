[
    {
      "name": "ApplicationA",
      "instance": [
        {
          "instanceId": "appA-instance-1",
          "hostName": "192.168.2.10",
          "ipAddr": "192.168.2.10",
          "status": "UP",
          "port": {
            "$": 8000,
            "@enabled": "true"
          },
          "securePort": {
            "$": 8443,
            "@enabled": "false"
          },
          "app": "ApplicationA",
          "metadata": {
            "version": "2.1.0",
            "zone": "eu-west-1a"
          }
        },
        {
          "instanceId": "appA-instance-2",
          "hostName": "192.168.2.11",
          "ipAddr": "192.168.2.11",
          "status": "DOWN",
          "port": {
            "$": 8001,
            "@enabled": "true"
          },
          "securePort": {
            "$": 8444,
            "@enabled": "false"
          },
          "app": "ApplicationA",
          "metadata": {
            "version": "2.1.0",
            "zone": "eu-west-1b"
          }
        },
        {
          "instanceId": "appA-instance-3",
          "hostName": "192.168.2.12",
          "ipAddr": "192.168.2.12",
          "status": "DOWN",
          "port": {
            "$": 8002,
            "@enabled": "true"
          },
          "securePort": {
            "$": 8444,
            "@enabled": "false"
          },
          "app": "ApplicationA",
          "metadata": {
            "version": "2.1.0",
            "zone": "eu-west-1b"
          }
        }
      ]
    },
    {
      "name": "ApplicationB",
      "instance": [
        {
          "instanceId": "appB-instance-1",
          "hostName": "192.168.3.10",
          "ipAddr": "192.168.3.10",
          "status": "DOWN",
          "port": {
            "$": 9000,
            "@enabled": "true"
          },
          "securePort": {
            "$": 9443,
            "@enabled": "false"
          },
          "app": "ApplicationB",
          "metadata": {
            "version": "1.2.0",
            "zone": "ap-southeast-1a"
          }
        }
      ]
    }
  ]