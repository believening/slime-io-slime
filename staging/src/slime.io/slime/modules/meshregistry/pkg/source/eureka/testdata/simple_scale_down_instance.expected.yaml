---
kind: ServiceEntry
apiVersion: networking.istio.io/v1alpha3
metadata:
  name: applicationa
  namespace: eureka
  labels:
    registry: eureka
  annotations: {}
spec:
  hosts:
    - applicationa
  ports:
    - number: 8000
      protocol: HTTP
      name: http-8000
  resolution: STATIC
  endpoints:
    - address: 192.168.2.10
      ports:
        http-8000: 8000
      labels:
        version: 2.1.0
        zone: eu-west-1a
---
kind: ServiceEntry
apiVersion: networking.istio.io/v1alpha3
metadata:
  name: applicationb
  namespace: eureka
  labels:
    registry: eureka
  annotations: {}
spec:
  hosts:
    - applicationb
  ports:
    - number: 9000
      protocol: HTTP
      name: http-9000
  resolution: STATIC
